### PR TITLE
Add "waitCommandIsSelected" method to the "CommandsExplorer" page-object in selenium tests

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Add "waitCommandIsSelected" method to the "CommandsExplorer" page-object and update "selectCommandByName" method with new check.
 
### What issues does this PR fix or reference?
Issue https://github.com/eclipse/che/issues/7364

### CODE REVIEW

**Review by tolusha — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsExplorer.java [hunk 1]
```diff
@@ -174,6 +174,22 @@ public void selectCommandByName(String commandName) {
...
+
+  /**
+   * wait until command will be selected
+   *
+   * @param commandName visible name of command in the command explorer
+   */
+  public void waitCommandIsSelected(String commandName) {
+    String locator =
+        String.format(
+            "//div[@id='command_%s' and contains(concat(' ', normalize-space(@class), ' '), ' selected')]",
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsExplorer.java]: Can we avoid "concat(' ', normalize-space(@class), ' ')" here by rewriting the xpath to the:
`//div[@id='command_%s' and contains(normalize-space(@class), 'selected')`
?

Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsExplorer.java]: On my regret no, in this case, this is the best way

Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsExplorer.java]: The simpler variant looks even better IMHO:
screenshot from 2017-11-20 17-37-19

**Comment by tolusha** [comment]
Do we need MySuite.xml ?

**Comment by tolusha** [comment]
Rename it respectively then

**Review by tolusha — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
Result of run tests
 https://ci.codenvycorp.com/view/qa/job/experimental-che-multiuser-integration-tests-che6/15/Selenium_tests_report/

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**